master/internal/api_trials.go

```diff
@@ -46,6 +46,7 @@ type TrialLogBackend interface {
 	AddTrialLogs([]*model.TrialLog) error
 	TrialLogCount(trialID int, filters []api.Filter) (int, error)
 	TrialLogFields(trialID int) (*apiv1.TrialLogsFieldsResponse, error)
+	DeleteTrialLogs(trialIDs []int) error
```

Comment:
random thought: Since the type is `TrialLogBackend`, the method names could probably all bear to have `Trial` removed from them. :stuck_out_tongue: